crates/router/src/core/payments/operations/payment_status.rs

```diff
@@ -397,7 +397,13 @@ pub async fn get_payment_intent_payment_attempt(
     merchant_id: &str,
     storage_scheme: enums::MerchantStorageScheme,
 ) -> RouterResult<(storage::PaymentIntent, storage::PaymentAttempt)> {
-    (|| async {
+    let le: error_stack::Result<
```

Comment:
Could we use a better variable name here (unless I'm missing out something obvious about what `le` stands for)?

Reply:
removed this as no longer needed